packages/daemon/src/app/controller/job.ts

```diff
@@ -17,6 +19,40 @@ export class JobController extends BaseEventController {
   @inject('pluginManager')
   pluginManager: PluginManager;
 
+  private async setupTracer(job: JobEntity) {
```

Comment:
What does this tracer for?

Reply:
This is for `updateParam` and `run` entry.